File: B/Python3/b008.py
max = 0
incom = []
s = input().rstrip().split(' ')
N = int(s[0])
M = int(s[1])
if M>0 and N>0 :
    for i in range(M):
        e = input().rstrip().split(' ')
        sum = 0
        for n in e:
            sum += int(n)
        incom.append(sum)
    for n in incom:
        if n>0:
            max += n
print(max)

File: B/Go/b008.go
package main

import (
  "bufio"
  "fmt"
  "os"
  "strconv"
)

var sc = bufio.NewScanner(os.Stdin)

func nextLine() string {
  sc.Scan()
  return sc.Text()
}

func nextInt() int {
  sc.Scan()
  i, e := strconv.Atoi(sc.Text())
  if e != nil {
    panic(e)
  }
  return i
}

func main() {

  sc.Split(bufio.ScanWords)

  max := 0
  N := nextInt()
  M := nextInt()
  incom := make([]int, M)

  if M>0 && N>0 {
    for i := 0; i < M; i++ {
      sum := 0
      for j := 0; j < N; j++ {
        sum += nextInt()
      }
      incom[i] = sum
    }
    for i := 0; i < M; i++ {
      n := incom[i]
      if n>0 {
        max += n
      }
    }
  }
  fmt.Println(max)
}
